app/javascript/components/party/Intro.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Layout, Row, Col, Typography, Avatar } from "antd";
+
+const { Title, Paragraph } = Typography;
+
+const Intro = ({ party }) => (
+  <Layout>
+    <Layout.Content className="party-intro">
+      <Row>
+        <Col span={24} lg={17}>
+          <Title level={2}>Descrição do Partido</Title>
+          {party.description}
+        </Col>
+        <Col span={24} lg={7} className="party-intro__spokesperson">
```

Comment:
Currently we do not have the party representative

Reply:
Alright. I'll remove it for now.